### PR TITLE
partially succeeded in applying useScrollLock

### PR DESCRIPTION
**Pull request by Kepmon**
So, the thing is, this useScrollLock thing seems to work perfectly with the Options API, except it doesn't unlock the screen back 😛

I can't understand why. I mean, I tried to log the value of this `scrollLock.value` and it switches between `true` and `false` as it's suppose to do when the nav is toggled on mobile devices. So, it's my understanding you unlock the screen by setting this value to false, which I did. So, am I missing something here?

### CODE REVIEW

**Review by aleksanderwalczuk — CHANGES_REQUESTED** [review]
I don't think this emit is needed, you can lock scroll directly in nav component.

**Comment by aleksanderwalczuk** [comment]
Seems that useScrollLock works only in composition.
in order to use it add following to nav component:
```js
  setup() {
    const el = ref(document.body) // notice that this is reactive
    const isLocked = useScrollLock(el)

    return { isLocked }
  },
```

**Comment by Kepmon** [comment]
It seems you're were right, now it works perfectly.

Too bad I needed to use the composition API but thank you for your help, anyway 😛

File: src/components/TheNav.vue [hunk 1]
```diff
@@ -128,11 +131,23 @@ export default {
     ...mapActions(useUserStore, ['signOut']),
     toggleNav(cb) {
       this.isNavShown = !this.isNavShown
+      
+      if (this.isNavShown) {
+        this.isLocked = true
+      } else {
+        this.isLocked = false
+      }
```
Comment (aleksanderwalczuk) [inline src/components/TheNav.vue]:
```suggestion
        this.isLocked = this.isNavShown
```

**Review by aleksanderwalczuk — CHANGES_REQUESTED** [review]
Little refacfor and we can move on 🙃

File: src/components/TheNav.vue (hunk 1 above)
Comment (Kepmon) [reply src/components/TheNav.vue]: 🙈

I could think of it yesterday but today no...

**Corrected**

**Review by aleksanderwalczuk — APPROVED** [review]
Lgtm

**Pull request merged**